.travis-ci.sh

```diff
@@ -105,16 +107,16 @@ on the github pull request.
 ------------------------------------------------------------------------
 EOF
   # check that Changes has been modified
-  git diff $TRAVIS_MERGE_BASE..$TRAVIS_PR_HEAD --name-only --exit-code Changes \
+  git diff "$TRAVIS_MERGE_BASE".."$TRAVIS_PR_HEAD" --name-only --exit-code Changes \
```

Comment:
The middle double-quotes are unnecessary - could this be changed to `"$TRAVIS_MERGE_BASE..$TRAVIS_PR_HEAD"`, please

Reply:
Done.